botocore/parsers.py

```diff
@@ -507,18 +507,18 @@ def _handle_timestamp(self, shape, value):
         return self._timestamp_parser(value)
 
     def _do_error_parse(self, response, shape):
-        body = json.loads(response['body'].decode(self.DEFAULT_ENCODING))
-        error = {"Error": {}, "ResponseMetadata": {}}
+        body = self._parse_body_as_json(response['body'])
+        error = {"Error": {"Message": '', "Code": ''}, "ResponseMetadata": {}}
```

Comment:
I think the other one I thought of was this: https://github.com/boto/botocore/pull/677. In this case, we catch the exception and log why it failed for future reference.

It looks like there is precedence on this so I'd be fine with having defaults.